lib/Parse/ParseDecl.cpp

```diff
@@ -1079,6 +1105,7 @@ static bool parseBaseTypeForQualifiedDeclName(Parser &P, TypeRepr *&baseType) {
 static bool parseQualifiedDeclName(Parser &P, Diag<> nameParseError,
                                    TypeRepr *&baseType,
                                    DeclNameRefWithLoc &original) {
+  {
```

Comment:
Could you please indent the code that's within curly braces, to make the nesting clearer?
I recommend using [`git clang-format`](https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/git-clang-format) to automatically format patches.

Reply:
Dan did this himself.  Thanks!